velox/common/base/tests/BitUtilTest.cpp

```diff
@@ -461,12 +461,27 @@ TEST_F(BitUtilTest, forEachBit) {
 }
 
 TEST_F(BitUtilTest, hash) {
-  std::unordered_set<size_t> hashes;
-  const char* text = "Forget the night, come live with us in forests of azure";
-  for (int32_t i = 0; i < strlen(text); ++i) {
-    hashes.insert(hashBytes(1, text, i));
+  std::unordered_map<uint64_t, int32_t> hashes;
+  std::string text =
+      "Forget the night, come live with us in forests of azure, "
+      "for we have constructed pyramids in honor of our escaping...";
+  for (int32_t i = 0; i < text.size(); ++i) {
+    auto hash = hashBytes(1, text.data(), i);
```

Comment:
Can we have a test calling `hashBytes` starting at `text.data()+1` as well to make sure unaligned load is working?